README.md

```diff
@@ -47,37 +59,41 @@ So that I am reassured that my order will be delivered on time
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+## Domain model
+![alt text](media/domain_model.png "Domain model for Takeaway Challenge")
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+# Reflections
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+## How I planned to solve this challenge
+*I drew on feedback from peer review and self-reflection on my [Airport Challenge programme](https://github.com/katemyoung/airport_challenge) to set goals for process improvement in this challenge.*
+- [x] Parse user stories and extract verbs and nouns into class diagram domain model
+- [x] Feature test the implementation of the first user story in irb `menu.list`
+- [x] Follow a tight TDD process of **red, green, refactor** to test drive the development of the feature for the first user story.
+- [x] Aim for 100% test coverage whilst refactoring previous unit tests as they became vacuous.
+- [x] Use rubocop for code quality.
+- [x] If I go off track, to use git to restore a previous working version. 
+- [ ] Add a SimpleCov test coverage badge to this README
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+## What I learnt during this challenge:
 
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+**Technical**
+- The RSpec `output` matcher and how to format a string with new lines using backslash instead of quotes.
+- Hash `fetch` method
+- Use of RSpecs `let` construct and `before` hook to minimise duplication of test setup code. 
 
+**TDD**
+- The importance of moving in baby steps and using the error messages to guide you.
+- Taking the time to feature test in irb before, during and after. 
 
-In code review we'll be hoping to see:
+## Questions
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/main/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+- I'm not sure I have initialized the dishes class in the most efficient way. How to initalize a hash automatically?
+- I wonder if a separate Dish class would be more SRP.
```

Comment:
It's ok to use a simple {:dish, :price} hash as a dish, for now.

Reply:
Thank you